File: 27-RemoveElement/27-RemoveElement.java
// Last updated: 7/8/2026, 3:43:21 PM
class Solution {
    public int removeElement(int[] nums, int val) {
        int st=0;
        for(int i=0;i<nums.length;i++){
            if(val!=nums[i]){
                nums[st]=nums[i];
                st++;
            }
        }
        return st;
        
    }
}

File: 27-RemoveElement/27-RemoveElement.js
// Last updated: 7/8/2026, 3:43:23 PM
/**
 * @param {number[]} nums
 * @param {number} val
 * @return {number}
 */
var removeElement = function(nums, val) {

    let i=0
    for(j=0;j<nums.length;j++){
        if(val!=nums[j]){
            nums[i]=nums[j]
            i++

        }
    }
    return i
   
    
    
};